solr_test.go

```diff
@@ -17,8 +17,7 @@ var _ = Describe("Solr Client", func() {
 	solrClient = solr.NewSolrZK("zk:2181", "solr", "solrtest")
 	locator = solrClient.GetSolrLocator()
 
-	var err error
-	err = solrClient.Listen()
+	var err error = solrClient.Listen()
```

Comment:
Why not make this `err := solrClient.Listen()` ? Is Listen returning a concrete error type?